VERSION

```diff
@@ -1 +1 @@
-0.14.7-dev
+0.14.7
```

Comment:
run make helm-build to update the helm chart

Reply:
updated.